pkg/artifactory/resource_artifactory_access_token_test.go

```diff
@@ -348,6 +348,47 @@ func TestAccAccessTokenMissingGroup(t *testing.T) {
 	})
 }
 
+const wildcardGroupGood = `
```

Comment:
Please format the string properly and include it in the test. This isn't a sharable string